ios/add_lottie_animations.sh

```
@@ -0,0 +1,207 @@
+import SwiftUI
+import Lottie
+
+/// RU: Компонент для Lottie анимаций FitChef
+/// EN: Component for FitChef Lottie animations
+struct LottieAnimationView: View {
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)